setup.js

```diff
@@ -0,0 +1,184 @@
+const readline = require('readline');
```

Comment:
I notice that this is repeated quite constantly, and be able placed into a function.

```js
/** @param {Error} err */
const = handleErr(err) => {
    if (error.code === 'ENOENT' || error.code === 'EEXIST') {
      console.error(error.message);
      // Try to continue execution even if some non-critical issues occur
    } else {
      console.error(error);
      process.exit(1);
    }
};

// ...

try {
  // ...
} catch (e) handleErr(e);
```

Reply:
Thank you for the suggestion, I applied that wherever possible.